Procfile

```diff
@@ -0,0 +1,2 @@
+web: bundle exec rake prod
```

Comment:
is web the right term? Isn't this more of a `host`? I know we're avoiding the term `master`